rte_ctrl.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/sh
+
+PFX="api/v1"
+
+CMD_ID_RELAY=0
+CMD_ID_RESET=8
+CMD_ID_POWER=9
+
+# TODO add host address as positional argument
+HOST=http://192.168.4.38:8000
+
+
+get() {
+    curl --silent -X GET $HOST/$PFX/$1
+}
+
+patch() {
+    curl -X PATCH $HOST/$PFX/$1 -d "$2"
+}
+
+put() {
+    curl -X PUT $HOST/$PFX/$1 -d "$2"
+}
+
+post_file() {
+    curl -X POST $HOST/$PFX/$1 -F "file=@$2"
+}
+
+delete() {
+    curl -X DELETE $HOST/$PFX/$1
+}
+
+post() {
+    curl -X POST $HOST/$PFX/$1
+}
+
+
+case $1 in
```

Comment:
It's better to assign it to variable like here: https://github.com/Dasharo/coreboot/blob/novacustom_ns5x/develop/build.sh#L135
So it can be reused later more easily